File: 0001-two-sum/0001-two-sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var dict = [Int: Int]()

        for (i, value) in nums.enumerated() {
            if let index = dict[target - value] {
                return [index, i]
            } else {
                dict[value] = i
            }
        }
    
        return []
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        for pointer_one in range(len(nums)):
            for pointer_two in range(pointer_one):

                if (nums[pointer_one] + nums[pointer_two]) == target:
                    return [pointer_one, pointer_two]
        
